mlrun/common/schemas/alert.py

```diff
@@ -202,6 +202,24 @@ def templates_differ(self, other):
         )
 
 
+class NotificationState(pydantic.BaseModel):
+    kind: str
+    status: Optional[str]
+
+
+class AlertActivation(pydantic.BaseModel):
+    name: str
+    project: str
+    severity: AlertSeverity
+    activation_time: datetime
+    entity_id: str
+    entity_kind: EventEntityKind
+    criteria: Optional[AlertCriteria]
```

Comment:
Why is it optional? What if value is nil?

Reply:
in a second though this field can never be None, since if you try to store an alert without a criteria then we assign it a default one, so the Optional can be removed.